fastpath/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Stage 1: Building
+FROM python:slim AS builder
```

Comment:
I would suggest we be a bit more specific with the image names used for making the docker file.

Let's put in here explicitly `3.13.3-slim-bookworm` in case future versions of python or debian cause issues with running the fastpath.